### DISCUSSION TITLE
Roadmap / White paper

### DISCUSSION THREAD

**Discussion by JulienRe** [thetaspere/theta#25] [Q&A] [answered]
Hello,

Where is the White Paper ? I don't found it.

Thank you.

**Comment by thetaspere** [comment]
We are still preparing the papers, please wait a little longer

**Comment by thetaspere** [comment, marked as answer]
Whitepapers available soon.
![Screenshot_1](https://user-images.githubusercontent.com/113691402/192109919-1b475f72-12fb-4344-80e1-7f84759efccf.jpg)
